src/Sarif.Viewer.VisualStudio/Telemetry/FeedbackTelemetryEvent.cs

```diff
@@ -42,6 +43,19 @@ public static void SendFeedbackTelemetryEvent(FeedbackModel feedback)
                 properties.Add(Snippet, JsonConvert.SerializeObject(feedback.Snippets));
             }
 
+            if (feedback.SarifLog != null)
+            {
+                // property has limit of 8192 on string length
+                int maxLength = 8192;
+                string sarifLog = JsonConvert.SerializeObject(feedback.SarifLog);
+                if (sarifLog.Length > maxLength)
+                {
+                    sarifLog = sarifLog.Substring(0, maxLength);
+                }
+
+                properties.Add(SarifLog, JsonConvert.SerializeObject(feedback.SarifLog));
```

Comment:
>JsonConvert.SerializeObject(feedback.SarifLog) [](start = 41, length = 46)

i should use sarifLog variable, since it already contains the converted json